### DISCUSSION TITLE
Changing burner height as a function of time

### DISCUSSION THREAD

**Discussion by Tbuttts21** [firemodels/fds#12164] [General]
I was doing a model of a large pool of flammable material. The height of the flammable material will decrease as a function of time. Historically, I have just assumed that the burner height stays at its maximum for the full duration of the simulation, but I was wondering if there was a way to change the Z value of the burner as a function of time. Is this a function in FDS? Is this something that might be added in the future?

**Comment by mcgratta** [comment]
There is a BURN_AWAY feature that would remove grid cells that represent the liquid. However, I do not recommend this because when a cell is removed, the temperature profile in depth is wiped out and started from ambient. 

Everything is averaged over the volume a grid cell, so slowly "eating" away a grid cell would not change the simulation.
